content/tutorials/iis.md

```diff
@@ -12,42 +12,66 @@ aliases: ["/docs/grafana/latest/tutorials/iis/"]
 
 # Use IIS with URL Rewrite as a Reverse Proxy
 
-If you want Grafana to be a subpath or subfolder under a website in IIS then the URL Rewrite module for ISS can be used to support this.
+If you want Grafana to be a subpath/subfolder under a website in IIS then the Application Request Routing (ARR) and URL Rewrite modules for ISS can be used to support this.
```

Comment:
```suggestion
To set Grafana as a subpath or subfolder under a website in IIS, use the Application Request Routing (ARR) and URL Rewrite modules for ISS for support.
```

Reply:
Hi Brenda, I deliberately changed `subpath or subfolder` to use a slash because they are the same thing.  Using `or` is misleading/misdirecting for the reader. otherwise other changes have no impact.